crates/iceberg/src/arrow/reader.rs

```diff
@@ -59,6 +62,12 @@ use crate::spec::{Datum, NestedField, PrimitiveType, Schema, Type};
 use crate::utils::available_parallelism;
 use crate::{Error, ErrorKind};
 
+/// Reserved field ID for the file path (_file) column per Iceberg spec
+pub(crate) const RESERVED_FIELD_ID_FILE: i32 = 2147483646;
+
+/// Column name for the file path metadata column per Iceberg spec
+pub(crate) const RESERVED_COL_NAME_FILE: &str = "_file";
```

Comment:
This is only pub for crate, so others won't be able to use it.

Reply:
Disregard the first comment. I see you do pub use in `scan/mod.rs`. But perhaps it should be in spec in the first place.